node/resolvers/Mutations/Users.ts

```diff
@@ -621,6 +623,17 @@ export const setCurrentOrganization = async (
       ctx
     )
 
+    const metricParams: ChangeTeamParams = {
```

Comment:
Similar question that I've been asking in other use cases: should we send the metric only when the request succeeds or for all requests? Here we're sending it only on success.

Reply:
Yes, that is the idea (I've also change the other PRs to send just on success)